lib/OpenQA/WebAPI/Controller/API/V1/Search.pm

```diff
@@ -163,6 +173,13 @@ sub _search_job_templates {
     return \@results;
 }
 
+sub _get_test_suite_ids {
+    my ($self, $keywords) = @_;
+    my $like            = {like => "%${keywords}%"};
+    my @test_suites_ids = map { $_->id } $self->schema->resultset('TestSuites')->search({name => $like})->all;
```

Comment:
Should we search the `description` as well here? 🤔

Reply:
I am not sure here, I thought only search the test suite name was enough. I don't want to get too many results. Users could do that in the `test suite` table.

Reply:
I would not do that right now as search results can become more cluttered. Good idea to think about later just in case we find better ways how to structure search results, optimize, optional filters, etc. You could note it down in the extension epic https://progress.opensuse.org/issues/70840